integrations/security/OCSF_1_1_0/readme.md

```diff
@@ -0,0 +1,100 @@
+    - OCSF 2003 - Compliance Finding
+    - OCSF 2004 - Detection Finding
+    - OCSF 2005 - Incident Finding
+  - Identity and Access Management
+    - OCSF 3001 - Account Change
+    - OCSF 3002 - Authentication
+    - OCSF 3003 - Authorize Session
+    - OCSF 3004 - Entity Management
+    - OCSF 3005 - User Access Management
+    - OCSF 3006 - Group Management
+  - Network Activity
+    - OCSF 4001 - Network Activity
+    - OCSF 4002 - HTTP Activity
+    - OCSF 4003 - DNS Activity
+    - OCSF 4004 - DHCP Activity
+    - OCSF 4005 - RDP Activity
+    - OCSF 4006 - SMB Activity
+    - OCSF 4007 - SSH Activity
+    - OCSF 4008 - FTP Activity
+    - OCSF 4009 - Email Activity
+    - OCSF 4010 - Network File Activity
+    - OCSF 4011 - Email File Activity
+    - OCSF 4012 - Email URL Activity
+    - OCSF 4013 - NTP Activity
+  - Discovery 
+    - OCSF 5001 - Device Inventory Info
+    - OCSF 5002 - Device Config State 
+    - OCSF 5003 - User Inventory Info
+    - OCSF 5004 - Operating System Patch State
+    - OCSF 5019 - Device Config State Change 
+  - Application Activity
+    - OCSF 6001 - Web Resources Activity
+    - OCSF 6002 - Application Lifecycle
+    - OCSF 6003 - API Activity
+    - OCSF 6004 - Web Resources Access Activity
+    - OCSF 6005 - Datastore Activity
+    - OCSF 6006 - File Hosting Activity
+    - OCSF 6007 - Scan Activity
+
+### OpenSearch Ingestion template
+The OpenSearch Ingestion template (`assets/OSI-pipeline.yaml`) provides a template you can use with an OpenSearch Ingestion pipeline to ingest OCSF data. 
+
+### Index State Management (ISM) policy
+The ISM policy (`assets/ISM.json`) rollsover the indexes daily or when they have reached 40GB. The ISM policy also deletes indexes that are more than 15 days old.
+
+### Initialization scripts
+The initialization scripts helps set up the component templates, index templates, ISM policy, and aliases in the OpenSearch cluster. 
+
+There are two scripts - one that uses basic auth (`assets/os_init_basic_auth.py`) - and one that uses IAM auth (`assets/os_init_IAM_auth.py`). 
+
+### OpenSearch objects
+The OpenSearch objects (`assets/OCSF_objects.ndjson`) contains visualizations, dashboards, and index patterns to help you get started with exploring OCSF data. Visualizations include: 
+
+* OCSF High level overview (All OCSF categories) page
+![OpenSearch Dashboard](static/all_ocsf_overview.png)
+
+* OCSF Findings (2000 series) overview page
+![OpenSearch Dashboard](static/ocsf_findings_overview_2000_series.png)
+
+* Network Activity (4001) Org level overview
+![OpenSearch Dashboard](static/ocsf_4001_overview.png)
+
+* Network Activity (4001) Account Level Drill Down
+![OpenSearch Dashboard](static/ocsf_4001_drilldown.png)
+
+* DNS Activity (4003) Org level overview
+![OpenSearch Dashboard](static/ocsf_4003_overview.png)
+
+## Installation instructions
+1. Download the index and component template zip files. Upload it to an S3 bucket or save it to your local machine.
```

Comment:
please create 2 workflows for instructions:
- S3 based
- file based

Reply:
for now, only security lake is supported due to the nature of the OCSF schema.  were probably going to have to get on a call to explain how this works...OCSF has roughly 43 class_uids under 6 broad categories.  the solutions covers all log flavors for these constructs...otherwise your building bazillion configs which breaks the concept here.  each OCSF class has an activity (such as network activity or DNS activity as noted in this reply comment.  there is no concept of Nginx or VPC flow logs or CloudTrail logs...and it feels like this solution for integrations is too narrowly scoped on products (which by the way are a filed in the OCSF spec).  at this point in time, there is no file based activity and there is no concept of delineating the products.  OCSF is a wrapper schema for all things security and I am going to raise an issue here shortly because the Integrations feature is not Gumby enough to make this work with the existing Integrations framework.  But we need a place for the index templates, visualizations, index patterns / saved objects, ISM templates, creation of aliases and rollovers with ISM...hopefully you get the point on the challenge here.  the things is that it is implemented with scripts because the lack of flexibility...so all the comments for each OCSF class are going to be the same from Kevin Low and I.  we can always push to AWS samples, but...i want this in Integrations...and with a few changes to the Integrations plugin, we are there.

Reply:
Thanks @kffallis for the detailed info
lets add this knowledge into the description and what is our goals for the evolution of integration being able to actually automate these steps